changelog/README.md

```diff
@@ -0,0 +1,67 @@
+Pending changelog
+=================
+
+This will get copied to dlang.org and cleared when master gets
+merged into stable prior to 2.072
+
+Please put your changelog entries into this folder.
+If you fix a bug fix, a changelog entry will automatically be generated.
+
+You can preview the changes with:
+
+```
+./build_changelog.d > doc.dd
+dmd -D doc.dd
+```
+
+Short changelog
+---------------
+
+Most users just want to quickly scan the changelog for areas of their interest,
+hence it's nice to provide a concise one-sentence information about your
+modification, it could look like this:
+
+```
+$(XREF range, padLeft) and $(XREF range, padRight) were added.
```

Comment:
Currently it just does the same as the previous changelog. 
Both short and long are put into lists, so it's possible to have links in there. However I do agree that creating links between them would be great. 
Ping @CyberShadow - would this change in functionality be okay?